Apps/Sandcastle/gallery/Geometry Height Reference.html

```diff
@@ -53,11 +67,63 @@
     }
 }]);
 
-var longitude = 6.850615989890521;
-var latitude = 45.89546589994886;
+var longitude = 6.950615989890521;
+var latitude = 45.79546589994886;
 var delta = 0.001;
 
-function addEntity(i, j) {
+function addGeometry(i, j) {
+    var west = longitude + delta * i;
+    var north = latitude + delta * j + delta;
+
+    var type = Math.floor(Math.random() * 3);
+    if (type === 0) {
+        viewer.entities.add({
+            position: Cesium.Cartesian3.fromDegrees(west, north, 0.0),
+            box : {
+                dimensions : new Cesium.Cartesian3(40.0, 30.0, 50.0),
+                material : Cesium.Color.fromRandom({alpha : 1}),
```

Comment:
The outlines look pretty bad to me because of the opaqunesss, can we set alpha .9 here and add `.withAlpha(0.9)` to the outlineCloud below? (or just get of outlines).

Reply:
I didn't like the way that looked, so I removed the outlines instead